luminance/src/tess.rs

```diff
@@ -269,6 +277,7 @@ impl<'a, C> TessBuilder<'a, C> {
       vert_nb: 0,
       instance_buffers: Vec::new(),
       inst_nb: 0,
+      patch_vert_nb: 0,
```

Comment:
I’m wondering. Isn’t this overlapping with `mode: Mode::Patch` + `vert_nb`?

Reply:
This controls the number of vertices per patch, and to my understanding, vert_nb controls the total number of vertices. So to render three quads, you would set vert_nb = 12 and patch_vert_nb = 4.

Reply:
Maybe I should elaborate on this in the documentation.

Reply:
Oh, yeah, you’re right. Yep, some doc on that would be neat!